libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CustomDropdownActions.tsx

```diff
@@ -28,13 +29,31 @@ function useIsConfigButtonVisible(filterDisplayFormRef: ObjRef) {
     const filterDisplayForm = dfMap.get(filterDisplayFormRef);
     invariant(filterDisplayForm);
 
-    const attrMap = useDashboardSelector(selectAllCatalogAttributesMap);
-    const filterAttribute = attrMap.get(filterDisplayForm.attribute);
+    const attributesMap = useDashboardSelector(selectAllCatalogAttributesMap);
+
+    const displayFormRef = useMemo(() => {
+        return [filterDisplayFormRef];
+    }, [filterDisplayFormRef]);
+
+    const { attributes } = useAttributes(displayFormRef);
+
+    if (!attributes) {
+        return null;
```

Comment:
Should probably return false (boolean - so it's consistent with other return types)?

Reply:
Yes, nice catch. Thanks